<!DOCTYPE html>
<!--[if IE 8]><html class="no-js lt-ie9" lang="en" > <![endif]-->
<!--[if gt IE 8]><!--> <html class="no-js" lang="en" > <!--<![endif]-->
<head>
  <meta charset="utf-8">
  <meta http-equiv="X-UA-Compatible" content="IE=edge">
  <meta name="viewport" content="width=device-width, initial-scale=1.0">
  
  
  
  <title>Use Case 3-3 - GalaxyKickstarter</title>
  

  <link rel="shortcut icon" href="../img/favicon.ico">

  
  <link href='https://fonts.googleapis.com/css?family=Lato:400,700|Roboto+Slab:400,700|Inconsolata:400,700' rel='stylesheet' type='text/css'>

  <link rel="stylesheet" href="../css/theme.css" type="text/css" />
  <link rel="stylesheet" href="../css/theme_extra.css" type="text/css" />
  <link rel="stylesheet" href="../css/highlight.css">

  
  <script>
    // Current page data
    var mkdocs_page_name = "Use Case 3-3";
    var mkdocs_page_input_path = "use_case_3-3.md";
    var mkdocs_page_url = "/use_case_3-3/";
  </script>
  
  <script src="../js/jquery-2.1.1.min.js"></script>
  <script src="../js/modernizr-2.8.3.min.js"></script>
  <script type="text/javascript" src="../js/highlight.pack.js"></script>
  <script src="../js/theme.js"></script> 

  
</head>

<body class="wy-body-for-nav" role="document">

  <div class="wy-grid-for-nav">

    
    <nav data-toggle="wy-nav-shift" class="wy-nav-side stickynav">
      <div class="wy-side-nav-search">
        <a href=".." class="icon icon-home"> GalaxyKickstarter</a>
        <div role="search">
  <form id ="rtd-search-form" class="wy-form" action="../search.html" method="get">
    <input type="text" name="q" placeholder="Search docs" />
  </form>
</div>
      </div>

      <div class="wy-menu wy-menu-vertical" data-spy="affix" role="navigation" aria-label="main navigation">
        <ul class="current">
          
            <li>
    <li class="toctree-l1 ">
        <a class="" href="..">Home</a>
        
    </li>
<li>
          
            <li>
    <li class="toctree-l1 ">
        <a class="" href="../about/">What is GalaxyKickstarter</a>
        
    </li>
<li>
          
            <li>
    <li class="toctree-l1 ">
        <a class="" href="../getting_started/">Getting started</a>
        
    </li>
<li>
          
            <li>
    <li class="toctree-l1 ">
        <a class="" href="../customizations/">Customizations</a>
        
    </li>
<li>
          
            <li>
    <li class="toctree-l1 ">
        <a class="" href="../installing tools and workflows/">Installing tools and workflows</a>
        
    </li>
<li>
          
            <li>
    <ul class="subnav">
    <li><span>Examples</span></li>

        
            
    <li class="toctree-l1 ">
        <a class="" href="../examples/docker/">Docker</a>
        
    </li>

        
            
    <li class="toctree-l1 ">
        <a class="" href="../examples/vagrant/">Vagrant</a>
        
    </li>

        
    </ul>
<li>
          
            <li>
    <li class="toctree-l1 ">
        <a class="" href="../faq/">Frequently asked questions</a>
        
    </li>
<li>
          
            <li>
    <li class="toctree-l1 ">
        <a class="" href="../available_roles/">Available roles</a>
        
    </li>
<li>
          
            <li>
    <li class="toctree-l1 ">
        <a class="" href="../available_variables/">Available variables</a>
        
    </li>
<li>
          
            <li>
    <li class="toctree-l1 ">
        <a class="" href="../editing_help/">Editing the readme</a>
        
    </li>
<li>
          
            <li>
    <ul class="subnav">
    <li><span>Metavisitor</span></li>

        
            
    <li class="toctree-l1 ">
        <a class="" href="../about_metavisitor/">About metavisitor</a>
        
    </li>

        
            
    <li class="toctree-l1 ">
        <a class="" href="../metavisitor_ansible/">Installing with GalaxyKickstarter</a>
        
    </li>

        
            
    <li class="toctree-l1 ">
        <a class="" href="../metavisitor_docker/">Installing with Docker</a>
        
    </li>

        
            
    <li class="toctree-l1 ">
        <a class="" href="../metavisitor_access_control/">Access and Control Metavisitor Galaxy instance</a>
        
    </li>

        
            
    <li class="toctree-l1 ">
        <a class="" href="../metavisitor_configure_references/">Prepare Metavisitor Galaxy instance for analyses</a>
        
    </li>

        
            
    <li class="toctree-l1 ">
        <a class="" href="../use_cases_input_data/">Prepare input data histories</a>
        
    </li>

        
            
    <li class="toctree-l1 ">
        <a class="" href="../use_case_1/">Use Cases 1-1 to 1-4</a>
        
    </li>

        
            
    <li class="toctree-l1 ">
        <a class="" href="../use_case_2/">Use Cases 2-1 and 2-2</a>
        
    </li>

        
            
    <li class="toctree-l1 ">
        <a class="" href="../use_case_3-1/">Use Case 3-1</a>
        
    </li>

        
            
    <li class="toctree-l1 ">
        <a class="" href="../use_case_3-2/">Use Case 3-2</a>
        
    </li>

        
            
    <li class="toctree-l1 current">
        <a class="current" href="./">Use Case 3-3</a>
        
            <ul>
            
                <li class="toctree-l3"><a href="#input-data-for-use-case-3-3">Input data for Use Case 3-3</a></li>
                
                    <li><a class="toctree-l4" href="#for-ebola-virus-samples">For Ebola virus samples:</a></li>
                
                    <li><a class="toctree-l4" href="#for-lassa-virus-samples">For Lassa virus samples:</a></li>
                
            
                <li class="toctree-l3"><a href="#history-for-use-case-3-3-ebola-virus">History for Use Case 3-3 / Ebola virus</a></li>
                
            
                <li class="toctree-l3"><a href="#history-for-use-case-3-3-lassa-virus-segment-l">History for Use Case 3-3 / Lassa virus, segment L</a></li>
                
            
                <li class="toctree-l3"><a href="#history-for-use-case-3-3-lassa-virus-segment-s">History for Use Case 3-3 / Lassa virus, segment S</a></li>
                
            
            </ul>
        
    </li>

        
    </ul>
<li>
          
            <li>
    <li class="toctree-l1 ">
        <a class="" href="../development/">Development</a>
        
    </li>
<li>
          
        </ul>
      </div>
      &nbsp;
    </nav>

    <section data-toggle="wy-nav-shift" class="wy-nav-content-wrap">

      
      <nav class="wy-nav-top" role="navigation" aria-label="top navigation">
        <i data-toggle="wy-nav-top" class="fa fa-bars"></i>
        <a href="..">GalaxyKickstarter</a>
      </nav>

      
      <div class="wy-nav-content">
        <div class="rst-content">
          <div role="navigation" aria-label="breadcrumbs navigation">
  <ul class="wy-breadcrumbs">
    <li><a href="..">Docs</a> &raquo;</li>
    
      
        
          <li>Metavisitor &raquo;</li>
        
      
    
    <li>Use Case 3-3</li>
    <li class="wy-breadcrumbs-aside">
      
    </li>
  </ul>
  <hr/>
</div>
          <div role="main">
            <div class="section">
              
                <h2 id="input-data-for-use-case-3-3">Input data for Use Case 3-3</h2>
<p>As for the previous Use Cases, the first step is to collect all input data in an history that we will name <code>Input data for Use Case 3-3</code>. </p>
<ul>
<li>Create a new history</li>
<li>Rename this history <code>Input data for Use Case 3-1</code></li>
<li>Using the tool <code>Extract reads in FASTQ/A format from NCBI SRA</code>, we are going to upload 63 paired end datasets. As explained in the previous chapter, the <code>Extract reads in FASTQ/A format from NCBI SRA</code> directly merges paired-end sequence data in a single file. As for the previous <code>Use Case 3-2</code>, collecting these 63 datasets is tedious. But the good news is that we are going to use repeatedly  the same <code>Extract reads in FASTQ/A format from NCBI SRA</code> tool and we only need in addition to change 63 times the datatype of these datasets from <code>fastq</code> to <code>fastqsanger</code>.</li>
</ul>
<p>So, here are the SSR ids to enter in the <code>Extract reads in FASTQ/A format from NCBI SRA</code> tool form (leaving <code>fastq</code> as an output format):</p>
<h5 id="for-ebola-virus-samples">For Ebola virus samples:</h5>
<p><code>SRR1613381</code>
<code>SRR1613377</code>
<code>SRR1613382</code>
<code>SRR1613378</code>
<code>SRR1613383</code>
<code>SRR1613379</code>
<code>SRR1613384</code>
<code>SRR1613380</code></p>
<p>When you have finishid with the 8 datasets, make sure to change their datatype from <code>fastq</code>to <code>fastqsanger</code>, and create a dataset collection (as explained in the previous chapter) of these 8 datasets that you will name <code>Ebola virus</code>.</p>
<h5 id="for-lassa-virus-samples">For Lassa virus samples:</h5>
<p><code>SRR1595772</code>
<code>SRR1595696</code>
<code>SRR1595665</code>
<code>SRR1595500</code>
<code>SRR1594619</code>
<code>SRR1595943</code>
<code>SRR1595673</code>
<code>SRR1595797</code>
<code>SRR1595763</code>
<code>SRR1595558</code>
<code>SRR1594664</code>
<code>SRR1595909</code>
<code>SRR1594651</code>
<code>SRR1595835</code>
<code>SRR1594698</code>
<code>SRR1613388</code>
<code>SRR1613389</code>
<code>SRR1613390</code>
<code>SRR1613391</code>
<code>SRR1613392</code>
<code>SRR1613393</code>
<code>SRR1613394</code>
<code>SRR1613395</code>
<code>SRR1613396</code>
<code>SRR1613397</code>
<code>SRR1613398</code>
<code>SRR1613399</code>
<code>SRR1595853</code>
<code>SRR1606288</code>
<code>SRR1613412</code>
<code>SRR1613403</code>
<code>SRR1606277</code>
<code>SRR1613386</code>
<code>SRR1613387</code>
<code>SRR1606267</code>
<code>SRR1614275</code>
<code>SRR1610580</code>
<code>SRR1595846</code>
<code>SRR1594606</code>
<code>SRR1606236</code>
<code>SRR1594723</code>
<code>SRR1594671</code>
<code>SRR1613414</code>
<code>SRR1613400</code>
<code>SRR1613401</code>
<code>SRR1613404</code>
<code>SRR1613402</code>
<code>SRR1613405</code>
<code>SRR1613407</code>
<code>SRR1613408</code>
<code>SRR1613409</code>
<code>SRR1613410</code>
<code>SRR1613406</code>
<code>SRR1613411</code>
<code>SRR1613413</code></p>
<p>When you have finishid with the 55 datasets, make sure to change their datatype from <code>fastq</code>to <code>fastqsanger</code>, and create a dataset collection (as explained in the previous chapter) of these 8 datasets that you will name <code>Lassa virus</code>.</p>
<ul>
<li>Copy the <code>vir1 nucleotide BLAST database</code> from the <code>References</code> history to the current history <code>Input data for Use Case 3-3</code>.</li>
</ul>
<h2 id="history-for-use-case-3-3-ebola-virus">History for Use Case 3-3 / Ebola virus</h2>
<ol>
<li>Stay in the history <code>Input data for Use Case 3-3</code></li>
<li>pick the workflow <code>Metavisitor: Workflow for Use Case 3-3</code> in the workflows menu. It is possible that the workflow manager complains about settings for the Trinity tool that is used in this workflow. This is a minor issue if happens: just edit the workflow, click on the tool <code>Trinity</code> and specify the number of processors accordingly to your computing infrastructure. Save the workflow and select the <code>run</code> option.</li>
<li>Before Step 1, you have to specify some parameters at run time. For Ebola virus, the field <code>target_virus</code> has to be filled with <code>Ebola</code> and the field <code>reference_virus</code> has to be filled with <code>NC_002549.1</code> (as a guide for reconstruction of the Ebola virus genome).</li>
<li>For Step 1, select <code>Ebola virus</code>.</li>
<li>For Step 2, select the <code>nucleotide vir1 blast database</code> (this should also be already selected)</li>
<li>As usual, check the box <code>Send results to a new history</code>, edit the name of the new history to <code>Use Case 3-3 Ebola virus</code>, and <code>Execute</code> the workflow ! Note, that for complex workflows with dataset collections in input, the actual warning that the workflow is started make take time to show up; you can even have a "504 Gateway Time-out" warning. This is not a serious issue: just go in your <code>User</code> -&gt; <code>Saved history</code> menu, you will see your <code>Use Case 3-3 Ebola virus</code> history running and you will be able to access it.</li>
</ol>
<p>The workflow for Use Case 3-3 may take a long time. Be patient.</p>
<h2 id="history-for-use-case-3-3-lassa-virus-segment-l">History for Use Case 3-3 / Lassa virus, segment L</h2>
<ol>
<li>Stay in the history <code>Input data for Use Case 3-3</code></li>
<li>pick the workflow <code>Metavisitor: Workflow for Use Case 3-3</code> in the workflows menu. It is possible that the workflow manager complains about settings for the Trinity tool that is used in this workflow. This is a minor issue if happens: just edit the workflow, click on the tool <code>Trinity</code> and specify the number of processors accordingly to your computing infrastructure. Save the workflow and select the <code>run</code> option.</li>
<li>Before Step 1, you have to specify some parameters at run time. For Lassa virus, the field <code>target_virus</code> has to be filled with <code>Lassa</code> and the field <code>reference_virus</code> has to be filled with <code>NC_004297.1</code> (as a guide for reconstruction of the segment L of the Lassa virus genome).</li>
<li>For Step 1, select <code>Lassa virus</code>.</li>
<li>For Step 2, select the <code>nucleotide vir1 blast database</code> (this should also be already selected)</li>
<li>As usual, check the box <code>Send results to a new history</code>, edit the name of the new history to <code>Use Case 3-3 Lassa virus segment L</code>, and <code>Execute</code> the workflow ! Note, that for complex workflows with dataset collections in input, the actual warning that the workflow is started make take time to show up; you can even have a "504 Gateway Time-out" warning. This is not a serious issue: just go in your <code>User</code> -&gt; <code>Saved history</code> menu, you will see your <code>Use Case 3-3 Lassa virus segment L</code> history running and you will be able to access it.</li>
</ol>
<p>The workflow for Use Case 3-3 may take a long time. Be patient.</p>
<h2 id="history-for-use-case-3-3-lassa-virus-segment-s">History for Use Case 3-3 / Lassa virus, segment S</h2>
<ol>
<li>Stay in the history <code>Input data for Use Case 3-3</code></li>
<li>pick the workflow <code>Metavisitor: Workflow for Use Case 3-3</code> in the workflows menu. It is possible that the workflow manager complains about settings for the Trinity tool that is used in this workflow. This is a minor issue if happens: just edit the workflow, click on the tool <code>Trinity</code> and specify the number of processors accordingly to your computing infrastructure. Save the workflow and select the <code>run</code> option.</li>
<li>Before Step 1, you have to specify some parameters at run time. For Lassa virus, the field <code>target_virus</code> has to be filled with <code>Lassa</code> and the field <code>reference_virus</code> has to be filled with <code>NC_004296.1</code> (as a guide for reconstruction of the segment S of the Lassa virus genome).</li>
<li>For Step 1, select <code>Lassa virus</code> (this should be already selected).</li>
<li>For Step 2, select the <code>nucleotide vir1 blast database</code> (this should also be already selected)</li>
<li>As usual, check the box <code>Send results to a new history</code>, edit the name of the new history to <code>Use Case 3-3 Lassa virus segment S</code>, and <code>Execute</code> the workflow ! Note, that for complex workflows with dataset collections in input, the actual warning that the workflow is started make take time to show up; you can even have a "504 Gateway Time-out" warning. This is not a serious issue: just go in your <code>User</code> -&gt; <code>Saved history</code> menu, you will see your <code>Use Case 3-3 Lassa virus segment S</code> history running and you will be able to access it.</li>
</ol>
<p>The workflow for Use Case 3-3 may take a long time. Be patient.</p>
              
            </div>
          </div>
          <footer>
  
    <div class="rst-footer-buttons" role="navigation" aria-label="footer navigation">
      
        <a href="../development/" class="btn btn-neutral float-right" title="Development">Next <span class="icon icon-circle-arrow-right"></span></a>
      
      
        <a href="../use_case_3-2/" class="btn btn-neutral" title="Use Case 3-2"><span class="icon icon-circle-arrow-left"></span> Previous</a>
      
    </div>
  

  <hr/>

  <div role="contentinfo">
    <!-- Copyright etc -->
    
  </div>

  Built with <a href="http://www.mkdocs.org">MkDocs</a> using a <a href="https://github.com/snide/sphinx_rtd_theme">theme</a> provided by <a href="https://readthedocs.org">Read the Docs</a>.
</footer>
	  
        </div>
      </div>

    </section>

  </div>

<div class="rst-versions" role="note" style="cursor: pointer">
    <span class="rst-current-version" data-toggle="rst-current-version">
      
      
        <span><a href="../use_case_3-2/" style="color: #fcfcfc;">&laquo; Previous</a></span>
      
      
        <span style="margin-left: 15px"><a href="../development/" style="color: #fcfcfc">Next &raquo;</a></span>
      
    </span>
</div>

</body>
</html>
